src/transformations.hpp

```diff
@@ -284,14 +288,19 @@ inline int dense_space_size(PDE<P> const &pde)
   return dense_space_size(pde.get_dimensions());
 }
 
+inline int dense_dim_size(int const &degree, int const &level)
```

Comment:
```suggestion
inline int dense_dim_size(int const degree, int const level)
```
`int` is smaller than a pointer